cors_filter.go

```diff
@@ -21,7 +21,11 @@ import (
 type CrossOriginResourceSharing struct {
 	ExposeHeaders  []string // list of Header names
 	AllowedHeaders []string // list of Header names
-	AllowedDomains []string // list of allowed values for Http Origin. An allowed value can be a regular expression to support subdomain matching. If empty all are allowed.
+	// AllowedDomains list of allowed values for Http Origin.
+	// An allowed value can be a regular expression to support subdomain matching.
+	// Non-regular expression values will be changed into an exact match: ^yourdomain.com$
```

Comment:
I'm not sure `^yourdomain.com$` is an intuitive "exact match". The `.` is still a wildcard. You'd have to do `fmt.Sprintf("^%s$", regexp.QuoteMeta(each))` to get an "exact match".

Reply:
agreed. `^www.amazon.com$` still matches `wwwqamazon.com` and `www.amazonqcom`.